src/elements/link-list/link-list.visual.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { html, type TemplateResult } from 'lit';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualRegressionFixture,
+} from '../core/testing/private.js';
+
+import './link-list.js';
+import '../link/block-link.js';
+
+describe(`sbb-link-list`, () => {
+  let root: HTMLElement;
+
+  const cases = {
+    negative: [false, true],
+    orientation: ['horizontal', 'vertical'],
+    size: ['xs', 's', 'm'],
+  };
+
+  const linksTemplate = (): TemplateResult => html`
+    <sbb-block-link href="#">Rückerstattungen</sbb-block-link>
```

Comment:
Probably the same also in the stories, but according to our coding standards we would like english textes. If you don't mind, can you change it? Content doesn't matter ;)

Reply:
Sure 👍  The stories were fine by the way